packages/vuetify/src/components/VTreeview/VTreeviewNode.ts

```diff
@@ -285,10 +317,17 @@ const VTreeviewNode = baseMixins.extend<options>().extend({
         class: {
           [this.activeClass]: this.isActive,
         },
+        attrs: {
+          'aria-control': this.ariaControls,
+          id: this.htmlId,
+        },
         on: {
-          click: () => {
+          click: (event: Event) => {
             if (this.openOnClick && this.hasChildren) {
               this.checkChildren().then(this.open)
+              this.$nextTick(function () {
```

Comment:
- Use an arrow function
- Is this supposed to be called after `checkChildren` resolves?